test/helper-functions.bats

```diff
@@ -29,12 +29,12 @@ function teardown_file() {
     skip 'this test must come first to reliably identify when to run setup_file'
 }
 
-@test "check helper-functions.sh: _sanitize_ipv4_to_subnet_cidr" {
-    run docker exec mail_helper_functions bash -c ". /usr/local/bin/helper-functions.sh; _sanitize_ipv4_to_subnet_cidr 255.255.255.255/0"
+@test "check helper functions (ssl): _sanitize_ipv4_to_subnet_cidr" {
```

Comment:
Not `ssl` related and not in the `ssl.sh` helper. This is in the `tcp_ip.sh`/`network.sh` helper.

```suggestion
@test "check helper function (network.sh): _sanitize_ipv4_to_subnet_cidr" {
```